packages/experiments/src/components/Chiclet/Chiclet.types.ts

```diff
@@ -37,6 +41,36 @@ export interface IChicletProps extends React.Props<ChicletBase> {
    */
   description?: React.ReactElement<JSX.Element>;
 
+  /**
+   * Image to render for the component.
+   */
+  image?: string;
+
+  /**
+   * Image Width to render for the component.
+   */
+  imageWidth?: string;
+
+  /**
+   * Image Height to render for the component.
+   */
+  imageHeight?: string;
```

Comment:
Can also be `number | string`

https://github.com/OfficeDev/office-ui-fabric-react/blob/e99af7f488bafb906dd34b79b2d9aeb5fb0accef/packages/office-ui-fabric-react/src/components/Image/Image.types.ts#L236